.gitignore

```diff
@@ -50,4 +50,6 @@ tenderly.yaml
 # Github #
 ##########
 
-secrets/
\ No newline at end of file
+secrets/
+
+types/
```

Comment:
These are meant to be temporary files generated for the tests so we are not committing them